windows/src/ext/sentry/Sentry.Client.pas

```diff
@@ -356,6 +358,18 @@ function TSentryClient.EventIDToString(AGuid: PByte): String;
   end;
 end;
 
+procedure TSentryClient.Breadcrumb(const BreadcrumbType, Message: string; const Category: string = ''; const Level: string = 'info');
+var
+  crumb: sentry_value_t;
+begin
+  crumb := sentry_value_new_breadcrumb(PAnsiChar(UTF8Encode(BreadcrumbType)), PAnsiChar(UTF8Encode(Message)));
+  if Category <> '' then
+    sentry_value_set_by_key(crumb, 'category', sentry_value_new_string(PAnsiChar(UTF8Encode(Category))));
+  if Level <> '' then
```

Comment:
Won't `Level` always be at least `'info'`?

Reply:
Can pass in `''` as a parameter, in which case that overrides `'info'`, and just didn't want to pass empty string in that case.